python/ray/experimental/channel/serialization_context.py

```diff
@@ -10,13 +10,23 @@ def __init__(self):
         self.torch_device: Optional["torch.device"] = None
         self.use_external_transport: bool = False
         self.tensors: List["torch.Tensor"] = []
+        self.data: Optional[Dict[str, Any]] = {}
```

Comment:
Remove `Optional` since this is never `None`?

Reply:
Updated https://github.com/ray-project/ray/pull/45591/commits/cc2e89739c1a9460416114f846ce09fe79b83c0c